packages/expo-ui/android/src/main/java/expo/modules/ui/PickerView.kt

```diff
@@ -38,7 +83,21 @@ class PickerView(context: Context, appContext: AppContext) : ExpoComposeView<Pic
                 onOptionSelected(mapOf("index" to index, "label" to label))
               },
               selected = index == selectedIndex,
-              label = { Text(label) }
+              label = { Text(label) },
+              colors = SegmentedButtonDefaults.colors(
+                activeBorderColor = colors.activeBorderColor.compose,
+                activeContentColor = colors.activeContentColor.compose,
+                inactiveBorderColor = colors.inactiveBorderColor.compose,
+                inactiveContentColor = colors.inactiveContentColor.compose,
+                disabledActiveBorderColor = colors.disabledActiveBorderColor.compose,
+                disabledActiveContentColor = colors.disabledActiveContentColor.compose,
+                disabledInactiveBorderColor = colors.disabledInactiveBorderColor.compose,
+                disabledInactiveContentColor = colors.disabledInactiveContentColor.compose,
+                activeContainerColor = colors.activeContainerColor.compose,
+                inactiveContainerColor = colors.inactiveContainerColor.compose,
+                disabledActiveContainerColor = colors.disabledActiveContainerColor.compose,
+                disabledInactiveContainerColor = colors.disabledInactiveContainerColor.compose
+              )
```

Comment:
I think it would be better to have a structure of a single directory per UI element, and then having files like
- `Component.kt` for the "actual" component code
- `ComponentProps.kt` \ `ComponentRecords.kt` for props, records, enums etc. (I'm not sure which name is more fitting)

In that case it would be cleaner to keep the `PickerColors` and `PickerProps` classes  there and have a conversion function that converts the provided props into `SegmentedButtonDefaults.colors` to avoid boilerplate in the "actual" component code.

Reply:
agreed, but I'd do the refactor as a separate PR ideally – I can follow up on this right after this one, or maybe you feel up for it? :)